### PR TITLE
postupload fixes

### PR DESCRIPTION
**Pull request by bikubi**
fcf2fdb855d94ac60a35b02eea83669d3b8168b7 is the meat here

### CODE REVIEW

**Comment by bikubi** [comment]
> [fcf2fdb](https://github.com/okfde/froide/commit/fcf2fdb855d94ac60a35b02eea83669d3b8168b7) is the meat here

...and `autoApprove` is not a great name...

File: frontend/styles/maps.scss [hunk 1]
```diff
@@ -61,5 +61,11 @@ $utilities: (
     responsive: true,
     property: position,
     values: static relative absolute fixed sticky
+  ),
+  'max-width': (
```
Comment (krmax44) [inline frontend/styles/maps.scss:65]: can't find these classes used anywhere. should we remove them then?

**Comment by krmax44** [comment]
@pajowu @stefanw maybe take a quick look at the `fullscreen_app.html` template I introduced, which required some changes to the base template. https://github.com/okfde/froide/pull/876/files#diff-f523c28415b61abcd34ee758b144095d2b5819a2cd1070590d8d4978ffaf1200

File: frontend/styles/maps.scss (hunk 1 above)
Comment (bikubi) [reply frontend/styles/maps.scss:65]: they're here: https://github.com/okfde/froide/blob/bikubi/postupload-fixes/frontend/javascript/components/redaction/pdf-redaction.vue#L164 `mw-lg-50 mw-xl-25`

**Pull request merged**